contracts/multisig-prover/src/encoding/abi/mod.rs

```diff
@@ -63,7 +63,7 @@ impl From<&WorkerSet> for WeightedSigners {
         WeightedSigners {
             signers,
             threshold,
-            nonce: Uint256::from(worker_set.created_at).to_le_bytes().into(),
+            nonce: Uint256::from(worker_set.created_at).to_be_bytes().into(),
```

Comment:
can you test coverage for this so changes are caught?

Reply:
tracked [here](https://axelarnetwork.atlassian.net/browse/AXE-3994)